packages/accordions/src/styled/timeline/StyledTimeline.ts

```diff
@@ -8,7 +8,7 @@
 import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
-const COMPONENT_ID = 'timeline.timeline';
+const COMPONENT_ID = 'timeline';
```

Comment:
☝️ Given that this component hasn't hit the website, it seemed safe to replace this non-standard naming scheme.